lib/common.js

```diff
@@ -764,6 +761,36 @@ function readApp(projectId, appId, cb) {
   }
 }
 
+function deleteApp(projectId, appId, cb) {
```

Comment:
[Question] Where this code came from? 

What's your plan for common commands? Do we leaving them in this folder for this PR or trying to integrate with fh3?